macos/FluentUI/Button/Button.swift

```diff
@@ -546,6 +591,18 @@ public enum ButtonStyle: Int, CaseIterable {
 	case borderless
 }
 
+/// Indicates the state of the button for providing different images
+@objc(MSFButtonState)
+public enum ButtonState: Int, CaseIterable {
```

Comment:
is there particular reason you made it iterable?

Reply:
no particular reason, just saw we use it in other enums, will remove it